testsuite/integration-arquillian/tests/other/mod_auth_mellon/src/test/java/org/keycloak/testsuite/mod_auth_mellon/ModAuthMellonTest.java

```diff
@@ -56,10 +57,12 @@ public void singleLoginAndLogoutTest() {
         modAuthMellonProtectedResourcePage2.navigateTo();
         URLAssert.assertCurrentUrlStartsWith(testRealmSAMLRedirectLoginPage);
 
+        setTimeOffset(SessionTimeoutHelper.IDLE_TIMEOUT_WINDOW_SECONDS * 1000);
```

Comment:
Should be enclosed in `try { setTimeOffset(...) ... } finally { resetTimeOffset(); }` block